install/kubernetes/cilium/values.yaml

```diff
@@ -341,7 +341,7 @@ global:
 
   # bandwidthManager enables BBR and FQ + EDT + BPF based rate-limiting for
   # Pods that are annotated with kubernetes.io/egress-bandwidth.
-  bandwidthManager: true
+  bandwidthManager: false
```

Comment:
We'll also need to update the K8sBandwidthTest tests to explicitly enable the bandwidth manager:
https://github.com/cilium/cilium/blob/d6ad56b02764f32db482c56e8dc41c42671a93eb/test/k8sT/Bandwidth.go#L146-L167